canonical-websites/www.ubuntu.com
# Fetch test of javascript based dropdown navigation

## Done

- Loading dropdowns with fetch onmouseover, one time only

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Test the dropdowns
- Test with no script

## Issues

Fixes #4527
